lib/testing-helpcenter/help_center_tags.rb

```diff
@@ -1,16 +1,26 @@
 # frozen_string_literal: true
 
+require_relative 'help_center_tags/articles_with_tag_popup'
+
 # Methods to work with tag list on https://helpcenter.onlyoffice.com/tags.aspx
 class HelpCenterTags
   include PageObject
 
   def initialize(instance)
     super(instance.webdriver.driver)
     @instance = instance
-    @xpath = "//div[contains(@class, 'div_for_index')]//a[contains(@class, 'text-container-link')]"
+    @tags_xpath = "//div[contains(@class, 'div_for_index')]//a[contains(@class, 'text-container-link')]"
   end
 
+  # @return [Array<String>] list of tags in tags page
   def tags_list
-    @instance.webdriver.get_text_of_several_elements(@xpath)
+    @instance.webdriver.get_text_of_several_elements(@tags_xpath)
+  end
+
+  def tag_click(tag_name)
+    tags_list_from_data = TagsListFromData.new
+    tag_index = (tags_list_from_data.tags.find_index(tag_name) + 1).to_s
```

Comment:
No point in using `to_s` in here since on next line `[#{tag_index}]`

and `#{variable}` construction in ruby always making to_s